include/mbedtls/ssl.h

```diff
@@ -851,6 +869,35 @@ typedef void mbedtls_ssl_async_cancel_t( mbedtls_ssl_context *ssl );
 #endif /* MBEDTLS_KEY_EXCHANGE_WITH_CERT_ENABLED &&
           !MBEDTLS_SSL_KEEP_PEER_CERTIFICATE */
 
+#if defined(MBEDTLS_SSL_DTLS_SRTP)
+
+#define MBEDTLS_TLS_SRTP_MAX_KEY_MATERIAL_LENGTH    60
```

Comment:
It is linked to the size of the key and salt needed by the most demanding SRTP profile available (currently AES128, there is another RFC to use higher profiles but it will be another PR).

Actually this define has nothing to do in ssl.h since the key material generation now takes place in a callback (implemented in testers programs ssl_client2 and ssl_server2) so I'll remove this define to put it there.

Reply:
Oh I see, thanks for explaining!